tests/mbed/host/ocalls.c

```diff
@@ -1,6 +1,11 @@
 // Copyright (c) Open Enclave SDK contributors.
 // Licensed under the MIT License.
 
+// Suppress warnings against using _open() instead of _sopen_s().
+#ifdef _WIN32
+#pragma warning(disable : 4996)
```

Comment:
If we're already disabling 4996, why bother with replacing the POSIX aliases with the Windows implementation-specific names (i.e. `_` prefixed names)?

Reply:
I tried `_sopen_s()` and it did not seem to work well so I just restored to the original version.